frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -32,18 +41,26 @@ const DataFormat: FC<Props> = ({
         name="dataFormat"
         disabled={isDisabled}
         onChange={handleChange}
-        selectedValue={format}
+        selectedValue={selectedFormat}
       >
         <Radio
           disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.H5AD)}
           label=".h5ad (AnnData v0.7)"
           value={DATASET_ASSET_FORMAT.H5AD}
         />
-        <Radio
-          disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.RDS)}
-          label=".rds (Seurat v3)"
-          value={DATASET_ASSET_FORMAT.RDS}
-        />
+        <Tooltip
+          disabled={!rdsSkipped}
```

Comment:
`rdsSkipped` is, for current purposes, behaving the same as `!availableFormats.includes(DATASET_ASSET_FORMAT.RDS`, right? I think we should just use the latter.

Reply:
I did this method because I don't want to incorrectly inform users that RDS assets are missing because of the sparce matrix limitations when that isn't the case.

If we don't pass down the skip status, **whenever** the RDS assets are missing we'll say that it is due to the sparce matrix

Reply:
Ok, sounds good 👍